### ISSUE TITLE
Nested folder structure

### ISSUE THREAD

**Issue by jrtcppv** [cvisionai/tator#1090]
This issue is to allow for media to have a nested folder structure just like a filesystem. This could be implemented by expanding or replacing the `Section` objects with an Ltree object. Combined with #374, this could allow for preserving on-disk directory structure when doing directory uploads.

**Comment by bctcvai** [comment]
Just to add some clarifying comments on LTREE. The usage could replace the section name with an LTREE or add an LTREE field to the Section table. Lookups and navigation could be done via the LTREE field with indirect association to media objects via the current uuid association.

```python
Psuedo code: (likely actually requiring some Q object shenanigans) 
s = Section.objects.filter(ltree_fancy_lookup)
media = Media.objects.filter(attributes->section_uuid__in=s.values('uuid'))
```

**Comment by bctcvai** [comment]
Docs to LTREE for reference: https://www.postgresql.org/docs/current/ltree.html

**Comment by bctcvai** [comment]
Backend only for 1.2

**Comment by bctcvai** [comment]
Removed the tag for migration  as its already been taken care of

**Comment by marktaipan-cvisionai** [comment]
Closed with 0ec12422a5b9080b0fd27a51d749b607841bf30e
